lib/puppet/provider/ini_setting/splunk.rb

```diff
@@ -13,7 +13,7 @@ def self.file_path
     File.join(@file_path, file_name)
   end
 
-  def self.set_file_path(path)
+  def self.set_file_path(path) # rubocop:disable Style/AccessorMethodName
```

Comment:
I wimped out here.  Somebody with more knowledge of the module and how the inifile providers work could have a go if they really wanted to.  Probably not worth it though.